packages/jest-cli/src/reporters/CoverageReporter.js

```diff
@@ -59,6 +72,37 @@ class CoverageReporter extends BaseReporter {
       `));
     }
 
+    this._checkThreshold(config);
+  }
+
+  _addUntestedFiles(config: Config, runnerContext: RunnerContext) {
+    if (config.collectCoverageFrom && config.collectCoverageFrom.length) {
+      const files = matchFilesWithGlobs(
+        runnerContext.hasteContext.moduleMap,
```

Comment:
ah, got it. I guess this is fine for now. I'll have to go in and refactor some stuff soon :)